src/components/Checkers/Cell/index.ts

```diff
@@ -1,3 +1,17 @@
-import Cell from './Cell';
+import { connect } from 'react-redux';
+import { chooseCell, doStep } from 'store/checkers/actions';
+import { getFieldItem, getPossibleStepsCells } from 'store/checkers/selectors';
+import { AppStateType } from 'store/rootReducer';
+import Cell, { IProps } from './Cell';
 
-export default Cell;
+const mapStateToProps = (state: AppStateType, { cellNumber }: IProps) => ({
```

Comment:
create IProps above in this file `IProps {cellNumber: number}` and pass `props: IProps`